.github/workflows/changelog.yml

```
@@ -1,12 +1,16 @@
+on:
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)